app/services/family_finances/attendee_report.rb

```diff
@@ -69,6 +77,10 @@ def stay_row(stay)
       row(stay.to_s, Stay::SingleAttendeeCost.call(stay: stay).total)
     end
 
+    def stay_row_without_unit(stay)
+      row(stay.to_s(without_unit: true), Stay::SingleAttendeeCost.call(stay: stay).total)
+    end
```

Comment:
If we pass the arg through the `stay_row` method then we could remove the need for a new method that mostly contains duplicate code:

```
def stay_row(stay, without_unit: false)
  row(stay.to_s(without_unit: without_unit), Stay::SingleAttendeeCost.call(stay: stay).total)
end
```